src/pages/EnablePayments/OnfidoStep.js

```diff
@@ -0,0 +1,154 @@
+                />
+                {
+                    this.canShowOnfido() ? (
+                        <Onfido
+                            sdkToken={this.props.walletOnfidoData.sdkToken}
+                            onUserExit={() => {
+                                Navigation.goBack();
+                            }}
+                            onSuccess={(data) => {
+                                activateWallet(CONST.WALLET.STEP.ONFIDO, {
+                                    onfidoData: JSON.stringify({
+                                        ...data,
+                                        applicantID: this.props.walletOnfidoData.applicantID,
+                                    }),
+                                });
+                            }}
+                        />
+                    ) : (
+                        <View style={[styles.mh5, styles.mb5, styles.flex1, styles.justifyContentBetween]}>
+                            {!this.props.walletOnfidoData.hasAcceptedPrivacyPolicy && (
+                                <>
+                                    <View style={styles.justifyContentCenter}>
+                                        <Text style={[styles.mb5]}>
+                                            {this.props.translate('onfidoStep.acceptTerms')}
+                                            <TextLink
+                                                href="https://onfido.com/facial-scan-policy-and-release/"
+                                            >
+                                                {this.props.translate('onfidoStep.facialScan')}
+                                            </TextLink>
+                                            {', '}
+                                            <TextLink
+                                                href="https://onfido.com/privacy/"
+                                            >
+                                                {this.props.translate('common.privacyPolicy')}
+                                            </TextLink>
+                                            {' and '}
+                                            <TextLink
+                                                href="https://onfido.com/terms-of-service/"
+                                            >
+                                                {this.props.translate('onfidoStep.termsOfService')}
+                                            </TextLink>
+                                            .
+                                        </Text>
+                                    </View>
+                                    <Button
+                                        success
+                                        text={this.props.translate('common.continue')}
+                                        isLoading={this.props.walletOnfidoData.loading}
+                                        onPress={() => {
```

Comment:
NAB:
```suggestion
                                        onPress={fetchOnfidoToken}
```

Reply:
Yeah, I think again here we don't want to be passing events as the first param to `fetchOnfidoToken()` so it's OK the way it was.